package.json

```diff
@@ -47,9 +48,10 @@
 		"serve-static": "^1.7.1"
 	},
 	"scripts": {
+		"lint": "eslint --no-ignore papaparse.js Gruntfile.js .eslintrc.js 'player/**/*.js' 'tests/**/*.js'",
```

Comment:
I think we can remove the player folder from lint and focus only on other files. 

Player is here only for local testing

Reply:
Done.